server-side/server.py

```diff
@@ -0,0 +1,31 @@
+"""This module responds to the REST invocations from client side app."""
```

Comment:
Also add, for current project (Client is Kotlin Android app) A little more description will be helpful

Reply:
```suggestion
"""This module responds to the REST invocations from the client-side app."""
```
Added little more description.